frontend/src/views/Project/CaPage/components/CaDetailsSection.tsx

```diff
@@ -1,4 +1,4 @@
-import { faCheck, faCopy } from "@fortawesome/free-solid-svg-icons";
+import { faCheck, faCopy, faPencil } from "@fortawesome/free-solid-svg-icons";
```

Comment:
what do you think about adding some sort of a button here to forward users to the docs for certificate templates? or maybe just an information tooltip?

Certificate template is an abstraction outside of CAs... it might be confusing for some why it's here 

<img width="955" alt="image" src="https://github.com/user-attachments/assets/75c2d4f7-de03-49c5-be8f-1d7979f3ddf7">

Reply:
Will do a separate PR to sweep over these; I think we can likely put subtitles under various sections in PKI to clarify them in general.